### DISCUSSION TITLE
Dashboard Unauthorized

### DISCUSSION THREAD

**Discussion by myhivedev** [deckhouse/deckhouse#3233] [Q&A] [answered]
Hello! Dashboard error: Unauthorized. In the other of Deckhouse Web Interfaces everything works. Help me please.
Best regards.
![dashboard](https://user-images.githubusercontent.com/17111765/205700727-09d16d1a-ea8f-4f4f-8e27-5907a25c3d60.png)

**Comment by nabokihms** [comment, marked as answer]
Hello, @myhivedev. Probably token passed to kube-apiserver is not accepted. You can find out why by checking kube-apiserver logs (namespace kube-system).

Usually it happens because of entries in the `/etc/hosts` file. See for more details https://github.com/deckhouse/deckhouse/issues/329

**Comment by myhivedev** [comment]
@nabokihms, thanks a lot! Yes, a have used /etc/hosts. When i removed records from /etc/hosts dashboard start working.
